packages/shared/src/netlifyCacheHandler.ts

```diff
@@ -0,0 +1,47 @@
+import { isDevelopmentFromPublishableKey } from './keys';
+
+/**
+ * Cache busting parameter for Netlify to prevent cached responses
+ * during handshake flows with Clerk development instances.
+ *
+ * Note: This query parameter will be removed in the "@clerk/clerk-js" package.
+ *
+ * @internal
+ */
+export const CLERK_NETLIFY_CACHE_BUST_PARAM = '__clerk_netlify_cache_bust';
+
+/**
+ * Prevents infinite redirects in Netlify's functions by adding a cache bust parameter
+ * to the original redirect URL. This ensures that Netlify doesn't serve a cached response
+ * during the handshake flow.
+ *
+ * The issue happens only on Clerk development instances running on Netlify. This is
+ * a workaround until we find a better solution.
+ *
+ * See {@link https://answers.netlify.com/t/cache-handling-recommendation-for-authentication-handshake-redirects/143969/1}
```

Comment:
The `{@link }` is only really used for linking modules/types, for URLs we can use below

```suggestion
 * See https://answers.netlify.com/t/cache-handling-recommendation-for-authentication-handshake-redirects/143969/1
```